cost/turbonomics/allocate_virtual_machines_recommendations/aws/turbonomics_allocate_virtual_machines.pt

```diff
@@ -0,0 +1,373 @@
+name "Turbonomics Allocate Virtual Machine Recommendations AWS"
+rs_pt_ver 20180301
+type "policy"
+short_description "Turbonomics policy for allocating RI coverage for virtual machines [README](https://github.com/flexera-public/policy_templates/tree/master/cost/turbonomics/allocate_virtual_machines_recommendations/aws) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+severity "low"
+category "Cost"
+default_frequency "daily"
+info(
+  version: "2.1",
+  provider: "AWS",
+  source: "Turbonomics",
+  service: "Compute",
+  policy_set: "Allocate VMs",
+  recommendation_type: "Rate Reduction",
+  publish: "false"
+)
+
+##################
+# Parameters   #
+##################
+
+parameter "auth_cookie" do
+  type "string"
+  label "Authorization Cookie"
+  no_echo true
+  description "authorization cookie pulled from manual source"
+end
+
```

Comment:
I remember ask from Tatian, Please pass url as param, Apply the same for all the policies

Reply:
By URL you mean this one?
![image](https://user-images.githubusercontent.com/54189123/220986145-c61a583e-22a8-4f3a-983e-bbb856c23e4f.png)

Reply:
@taterdog590 - Could you please confirm?

Reply:
Yes, this value should be passed as parameter